gax-java/gax/src/test/java/com/google/api/gax/batching/BatcherImplTest.java

```diff
@@ -885,20 +888,35 @@ public void run() {
       } while (batcherAddThreadHolder.isEmpty()
           || batcherAddThreadHolder.get(0).getState() != Thread.State.WAITING);
 
+      long beforeGetCall = System.currentTimeMillis();
       executor.submit(
           () -> {
             try {
               Thread.sleep(throttledTime);
+              logger.info("Calling flowController.release");
```

Comment:
Can we lower the logging level so that the logs are not generated by default?

Reply:
I want these logs in case the problem recurs.

Reply:
I will remove these logs in February, if the flaky test does not recur.

Reply:
SGTM.

Reply:
https://github.com/googleapis/sdk-platform-java/pull/2519